### PR TITLE
Created Saudi Arabia parser and update it in zones (SA)

### PR DESCRIPTION
**Pull request by Suwailem1**
I crated a new parser for Saudi Arabia with name SA.py
I updated the Capacity in zones.json

### CODE REVIEW

File: config/zones.json [hunk 1]
```diff
@@ -4974,8 +4974,22 @@
...
+      "geothermal": 0
+    },
     "contributors": [
-      "https://github.com/q--"
+      "https://github.com/q--",
+      "https://github.com/Suwailem1"
+
     ],
     "parsers": {
       "consumption": "GCCIA.fetch_consumption"
```
Comment (madsnedergaard) [inline config/zones.json:4995]: You should add the new parser here under "production" :)

Comment (Suwailem1) [reply config/zones.json]: Done.

File: config/zones.json [hunk 2]
```diff
@@ -4974,11 +4974,27 @@
...
     "contributors": [
-      "https://github.com/q--"
+      "https://github.com/q--",
+      "https://github.com/Suwailem1"
+
     ],
     "parsers": {
-      "consumption": "GCCIA.fetch_consumption"
+      "consumption": "GCCIA.fetch_consumption",
+      "production": "GCCIA.fetch_production"
```
Comment (madsnedergaard) [inline config/zones.json:4996]: This should be `SA`, so it uses the new parser :)

```suggestion
      "production": "SA.fetch_production"
```

**Review by madsnedergaard — COMMENTED** [review]
Running the parser fails - you can test it yourself with `poetry run test_parser SA production`

Screenshot 2021-11-08 at 09 16 28

File: config/zones.json (hunk 2 above)
Comment (Suwailem1) [reply config/zones.json:4996]: production and consumption, not just production correct?

Comment (madsnedergaard) [reply config/zones.json]: Yeah, seems like the GCCIA parser isn't working anyway - so would make sense to use your's for both :)

**Comment by madsnedergaard** [comment]
Hi, I still can't get the parser to run - even if I fix the URL. Did the page content change? I can't find any consumption info the website we're scraping :)

**Comment by madsnedergaard** [comment]
Hey @Suwailem1 , how are things here - did you have a chance to look at it? :)

**Review by madsnedergaard — CHANGES_REQUESTED** [review]
Changing to "changes requested" to give better overview of the status :)

**Comment by madsnedergaard** [comment]
Unfortunately @Suwailem1 won't have time to work on this, so I'll close it for now as I couldn't find the required data.

**Pull request closed without merging**